content/en/docs/duplicate-certificate-limit.md

```diff
@@ -23,6 +24,31 @@ set" of hostnames for your certificate would be `[example.com]`. This rate limit
 is exceeded when a subscriber requests a certificate for the same "exact set" of
 hostnames more than 5 times in a single week.
 
+## Wait for Reset
+
+Revoking the previously issued certificates will not reset the Duplicate
+Certificate limit. Subscribers can request another certificate 168 hours (7
+days) after the  most recent successful issuance. If you're unsure of the date
+and time of your last successfully issued certificate, follow the directions
+below:
+
+- Open [crt.sh](https://crt.sh/).
```

Comment:
I don't think we should go into too much detail here. Crt.sh has certificates and pre-certificates that could be listed in any order but often all carts are listed most recently issued to oldest. As such, the top certificate in the view does not denote the beginning of the window.